src/Apps/Auction/Routes/__tests__/ConfirmBid.test.tsx

```diff
@@ -17,15 +21,26 @@ jest.mock("sharify", () => ({
   },
 }))
 
-const mockLocation: any = {
+const mockLocation: Partial<Location> = {
   search: "",
 }
 
-const setupTestEnv = () => {
+const mockDefaultData = (
+  data: DeepPartial<ConfirmBidQueryResponse> = {}
+): ConfirmBidQueryResponse =>
+  deepMerge<ConfirmBidQueryResponse, DeepPartial<ConfirmBidQueryResponse>>(
+    ConfirmBidQueryResponseFixture,
+    data
+  )
+
+const setupTestEnv = ({
+  location = mockLocation,
+  data = {},
+}: { location?: Partial<Location>; data?: any } = {}) => {
   return createTestEnv({
     TestPage: ConfirmBidTestPage,
     Component: (props: routes_ConfirmBidQueryResponse) => (
-      <ConfirmBidRouteFragmentContainer location={mockLocation} {...props} />
+      <ConfirmBidRouteFragmentContainer location={location as any} {...props} />
```

Comment:
Minor, but does this have to be `any` instead of a `Location`, as we did in the require login PR? 

https://github.com/artsy/reaction/pull/2936/files#diff-42d3becca5be832d10c7b1e055bef46dR40

Reply:
I could do that and was going to just now, but on second look my argument against is is that this isn't a _real_ location object, it only has a `search` value for the query string. I think the `any` does a better job communicating to the dev that this is not a `Location` - it will jump out at you when a test mysteriously breaks later - while still satisfying the type checker.

Reply:
Update: after talking with folks about Ts and testing philosophies I adopted your change. Thanks for raising!